### PR TITLE
Implement PostgreSQL

### PR DESCRIPTION
**Pull request by jonatasbaldin**
Implement PostgreSQL in Docker Compose using dj-database-url packge.

Related to #4.

@CheesecakeLabs/backend

### CODE REVIEW

**Comment by jonatasbaldin** [comment]
💃

File: docker-compose.yml [hunk 1]
```diff
@@ -2,8 +2,19 @@ version: '2'
...
       - "8000:8000"
+
+  db:
+    image: postgres:latest
+    ports:
+      - "5432:5432"
+    volumes:
+      - "./docker/db/pgdata:/var/lib/postgresql/data"
+    environment:
+      POSTGRES_PASSWORD: 'postgres'
```
Comment (smaniotto) [inline docker-compose.yml]: Do we need the quotes?

File: docker/web/check_db.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import socket
+import time
+import argparse
+""" Check if port is open, avoid docker-compose race condition """
```
Comment (smaniotto) [inline docker/web/check_db.py:4]: Module docstring should come first than imports, no?

File: docker/web/check_db.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+parser = argparse.ArgumentParser(description='Check if port is open, avoid\
+                                 docker-compose race condition')
```
Comment (smaniotto) [inline docker/web/check_db.py:7]: I will leave this comment for the future: regarding code style decision, will we break long/multiple arguments like this?

File: project_name/settings.py [hunk 4]
```diff
@@ -12,6 +12,8 @@
 
 import os
 
+import dj_database_url
+
```
Comment (smaniotto) [inline project_name/settings.py:16]: One extra blank line between imports and code?

File: project_name/settings.py [hunk 5]
```diff
@@ -119,3 +121,6 @@
 # https://docs.djangoproject.com/en/1.10/howto/static-files/
 
 STATIC_URL = '/static/'
+
+# Third-part services
+DATABASES['default'] = dj_database_url.parse('postgres://postgres:postgres@db:5432/postgres')
```
Comment (smaniotto) [inline project_name/settings.py:127]: This database url should come form env vars right?

File: docker-compose.yml (hunk 1 above)
Comment (jonatasbaldin) [reply docker-compose.yml]: They dont hurt :(

File: docker/web/check_db.py (hunk 2 above)
Comment (jonatasbaldin) [reply docker/web/check_db.py:4]: This script is temporary, will be improved at #15

File: docker/web/check_db.py (hunk 3 above)
Comment (jonatasbaldin) [reply docker/web/check_db.py:7]: This script is temporary, will be improved at #15

File: project_name/settings.py (hunk 4 above)
Comment (jonatasbaldin) [reply project_name/settings.py:16]: Separating the standard Django library and third-party packages.

Do you think we need a comment over there?

File: project_name/settings.py (hunk 5 above)
Comment (jonatasbaldin) [reply project_name/settings.py:127]: Yep, will be improved at #13

File: project_name/settings.py (hunk 4 above)
Comment (smaniotto) [reply project_name/settings.py:16]: I meant about having two lines between the imports and "code" start. I like separating the import into: standard, third-party and our-own :)

File: docker-compose.yml (hunk 1 above)
Comment (smaniotto) [reply docker-compose.yml]: They seem off, because of the single vs double-quotes. But ok, they don't hurt.

Comment (jonatasbaldin) [reply docker-compose.yml]: Added double quotes.

**Review by smaniotto — APPROVED** [review]

**Pull request merged**